src/test/java/io/jenkins/plugins/opentelemetry/api/ReconfigurableOpenTelemetryTest.java

```diff
@@ -94,6 +98,24 @@ void test_configuration_through_passed_properties_overwrites() {
         }
     }
 
+        @Test
+    void test_configuration_without_endpoint() {
+        try {
+            Map<String, String> otelConfig = new HashMap<>();
+            otelConfig.put("otel.instrumentation.jdbc.enabled", "true");
+            otelConfig.put("otel.traces.exporter", "otlp");
```

Comment:
`otlp`  or empty string `""` are synonyms, should we have both test cases?

Reply:
In that case, the issue is that the default value for `otel.traces.exporter` should be `otpl` not `""`.